src/LibLog/LibLog.cs

```diff
@@ -1823,7 +1824,7 @@ internal static class LogMessageFormatter
         /// <returns></returns>
         public static Func<string> SimulateStructuredLogging(Func<string> messageBuilder, object[] formatParameters)
         {
-            if(formatParameters == null)
+            if(formatParameters == null || !formatParameters.Any())
```

Comment:
Yeah micro! ;)

I did get slightly chastised previously for not being perfomance pedantic.

Thanks, will merge.
On 4 Mar 2015 10:02, "Erwin van der Valk" <notifications@github.com> wrote:

> In src/LibLog/LibLog.cs
> <https://github.com/damianh/LibLog/pull/39#discussion_r25760098>:
>
> > @@ -1823,7 +1824,7 @@ internal static class LogMessageFormatter
> >          /// <returns></returns>
> >          public static Func<string> SimulateStructuredLogging(Func<string> messageBuilder, object[] formatParameters)
> >          {
> > -            if(formatParameters == null)
> > +            if(formatParameters == null || !formatParameters.Any())
>
> Just did it. Shaved off 0.6 ticks for every log message written! :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/damianh/LibLog/pull/39/files#r25760098>.
>